helm/values.yaml

```diff
@@ -0,0 +1,39 @@
+common:
+  fullnameOverride: "pr-custom-review"
+  image:
+    repository: paritytech/pr-custom-review
+  serviceAccount:
+    create: false
+  podSecurityContext:
+    fsGroup: 1000
+    runAsNonRoot: true
+    runAsUser: 1000
+    runAsGroup: 1000
+  env:
+    LOG_FORMAT: json
+    PORT: 3000
+  secrets:
+    GITHUB_ACCESS_TOKEN: ref+vault://kv/cicd/gitlab/parity/mirrors/pr-custom-review#GITHUB_ACCESS_TOKEN
```

Comment:
Tried this PR

https://gitlab.parity.io/parity/mirrors/pr-custom-review/-/jobs/1578420#L45

complains about something related to base64

Reply:
Well probably we hit this issue: https://stackoverflow.com/questions/53394973/cant-create-secret-in-kubernetes-illegal-base64-data-at-input

Reply:
just base64 encoded the secret, please try one more time.

Reply:
> just base64 encoded the secret, please try one more time.

Didn't work https://gitlab.parity.io/parity/mirrors/pr-custom-review/-/jobs/1578629#L44